.github/workflows/pull-request.yml

```diff
@@ -126,6 +126,12 @@ jobs:
 
       - name: Generate Refresh Token Secret
         run: echo "REFRESH_TOKEN_SECRET=$(openssl rand -hex 32)" >> $GITHUB_ENV
+      
+      - name: Set up Transaction Logs path
+        run: |
+          mkdir -p logs
+          touch logs/transaction.log
+          echo "TRANSACTION_LOG_PATH=${GITHUB_WORKSPACE}/logs/transaction.log" >> $GITHUB_ENV
```

Comment:
1. Make the transaction log location a part of the `.env` file configuration
2. Update the `.env.sample` file with this information
3. Update `setup.ts` accordingly